packages/plans-grid-next/src/style.scss

```diff
@@ -707,42 +703,19 @@ $mobile-card-max-width: 440px;
 	}
 }
 
-/**
- * TODO clk migrate out of plans-grid-next
- */
-body.is-section-stepper,
-body.is-section-signup.is-white-signup {
-	.step-wrapper__header {
-		margin: 24px 20px 38px;
-		@include plans-2023-break-small {
-			margin: 24px 20px;
-		}
-	}
-}
-
-/**
- * TODO clk migrate out of plans-grid-next
- */
-.formatted-header {
-	margin-bottom: 0;
-}
-
-.plans-features-main__comparison-grid-container {
-	.plan-comparison-grid {
-		.plan-comparison-grid__feature-group-row.is-storage-feature {
-			align-items: flex-start;
-		}
+.plan-comparison-grid {
+	.plan-comparison-grid__feature-group-row.is-storage-feature {
+		align-items: flex-start;
 	}
-}
 
-.plan-comparison-grid {
 	.plan-features-2023-gridrison__actions {
 		margin-bottom: auto;
+	}
 
-		&-buttons {
```

Comment:
This pattern makes it hard to search & debug things. You cannot reach the full class from the browser in the editor. I'd avoid like fire 😬